### PR TITLE
Add emergency ward entrance

### PR DESCRIPTION
**Pull request by dvdmrtnz**
Added an icon for `emergency=emergency_ward_entrance` based on the  `entrance=main` icon. Also fixed the entrance icons to fit the pixel grid.

* Fixes https://github.com/organicmaps/organicmaps/issues/6777

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -16556,3 +16556,44 @@ uk:Соціальна установа
...
+ru:Вход в отделение неотложной помощи
+sk:Vchod do pohotovostného oddelenia
+sv:Ingång till akutmottagning
+sw:Mlango wa Wadi ya Dharura
+th:ทางเข้าแผนกฉุกเฉิน
+tr:Acil Servis Girişi
+uk:Вхід до відділення невідкладної допомоги
+vi:Lối vào khu cấp cứu
+zh-Hans:急诊室入口
+zh-Hant:急診病房入口
```
Comment (vng) [inline data/categories.txt:16599]: Honestly, I have doubts here. Please, remove it for now.

**Review by vng — CHANGES_REQUESTED** [review]

File: data/categories.txt (hunk 1 above)
Comment (dvdmrtnz) [reply data/categories.txt]: Removed

**Comment by vng** [comment]
Please, rebase!

**Comment by dvdmrtnz** [comment]
Rebased

**Review by vng — APPROVED** [review]

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**